src/st/class-wpml-page-builders-app.php

```diff
@@ -43,7 +43,15 @@ public function load_integration() {
 		if ( $factories ) {
 			foreach ( $factories as $factory ) {
 				$integration = $factory->create();
-				$integration->add_hooks();
+
+				if ( is_array( $integration ) ) {
+					foreach ( $integration as $hooks ) {
+						$hooks->add_hooks();
+					}
```

Comment:
@strategio I think it's better if:
- You move line 55 belove line 52
- Add `$this->integrations[] = $hooks;` below line 49